Pipfile

```diff
@@ -10,6 +10,7 @@ bandit = "*"
 click = "*"
 thoth-common = "*"
 thoth-analyzer = "*"
+thoth-python = {editable = true, path = "/home/kpostlet/project/python"}
```

Comment:
should be thoth-python = "*"?

Reply:
Oops, my bad. left over from local testing :/